ocrd_segment/ocrd-tool.json

```diff
@@ -291,6 +291,27 @@
           "default": "",
           "description": "Comma-separated list of forbidden image features (e.g. `binarized,despeckled`)."
         },
+        "classes": {
+          "type": "string",
+          "enum": [
+            "NoiseRegion",
+            "LineDrawingRegion",
+            "AdvertRegion",
+            "ImageRegion",
+            "ChartRegion",
+            "MusicRegion",
+            "GraphicRegion",
+            "UnknownRegion",
+            "CustomRegion",
+            "SeparatorRegion",
+            "MathsRegion",
+            "TextRegion",
+            "MapRegion",
+            "ChemRegion",
+            "TableRegion"
```

Comment:
I'm afraid this does not work (and is not valid). With `enum` you enforce any if these values, but thereby also prohibit combining them with commas.

If you want to enforce this syntactically, you'd have to set `type=array`, and then place an `enum` on its `items`. (But then, of course, the code would have to be adapted as well (by removing comma-splitting).

```suggestion
```